helix-core/src/test/java/org/apache/helix/integration/task/TestTaskSchedulingTwoCurrentStates.java

```diff
@@ -138,6 +139,17 @@ public void testTargetedTaskTwoCurrentStates() throws Exception {
     JobQueue.Builder jobQueue = TaskTestUtil.buildJobQueue(jobQueueName);
     jobQueue.enqueueJob("JOB0", jobBuilder0);
 
+    // Make sure master has been correctly switched to Participant1
+    boolean isMasterSwitchedToCorrectInstance = TestHelper.verify(() -> {
+      ExternalView externalView = _gSetupTool.getClusterManagementTool().getResourceExternalView(CLUSTER_NAME, DATABASE);
+      Map <String, String> stateMap = externalView.getStateMap(DATABASE + "_0");
```

Comment:
Is externalView possible to be null?

Reply:
Yeah it is possible. I added new check for the null EV.